src/main/groovy/io/seqera/wave/service/persistence/postgres/DbInitService.groovy

```diff
@@ -0,0 +1,88 @@
+/*
+ *  Wave, containers provisioning service
+ *  Copyright (c) 2023-2024, Seqera Labs
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.seqera.wave.service.persistence.postgres
+
+import java.sql.SQLException
+
+import groovy.transform.CompileStatic
+import io.micronaut.data.jdbc.runtime.JdbcOperations
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+
+/**
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@Singleton
+@CompileStatic
+class DbInitService {
+
+    @Inject
+    private JdbcOperations jdbcOperations
+
+    final static private String ddl = '''
+            CREATE TABLE IF NOT EXISTS wave_build (
+                id TEXT PRIMARY KEY,
+                data JSONB NOT NULL,
+                created_at TIMESTAMP DEFAULT NOW()
+            );
+
+            CREATE TABLE IF NOT EXISTS wave_request (
+                id TEXT PRIMARY KEY,
+                data JSONB NOT NULL,
+                created_at TIMESTAMP DEFAULT NOW()
+            );
+
+            CREATE TABLE IF NOT EXISTS wave_scan (
+                id TEXT PRIMARY KEY,
+                data JSONB NOT NULL,
+                created_at TIMESTAMP DEFAULT NOW()
+            );
```

Comment:
Question is this timestamp supplied or generated using the default value, if it's the later there may be an optimization we can do in Postgres using  generated columns 

This removes the need for additional JSON parsing at write where we can have an additional benefit of indexing

For example.

```
CREATE TABLE IF NOT EXISTS wave_scan (
    id TEXT PRIMARY KEY,
    data JSONB NOT NULL,
    created_at TIMESTAMP DEFAULT NOW(),
    creation_time TIMESTAMP GENERATED ALWAYS AS ((data->>'creationTime')::timestamp) STORED
);
```

The reference for this is https://www.postgresql.org/docs/current/ddl-generated-columns.html